src/control/control.h

```diff
@@ -198,14 +201,12 @@ class ControlDoublePrivate : public QObject {
 
     QAtomicPointer<ControlObject> m_pCreatorCO;
 
+    // name of the key to track using stats framework, unless the trackKey isNull().
     QString m_trackKey;
```

Comment:
```suggestion
    // name of the key to track using stats framework, unless the trackKey isNull().
    QString m_trackingKey;
```
I think this will help readability, because it would make it immediately visible, that no music track is meant here.

Reply:
Done